src/main/java/net/sf/jabref/gui/JabRefFrame.java

```diff
@@ -483,6 +483,7 @@ public void actionPerformed(ActionEvent e) {
     private final List<Object> openDatabaseOnlyActions = new LinkedList<>();
     private final List<Object> severalDatabasesOnlyActions = new LinkedList<>();
     private final List<Object> openAndSavedDatabasesOnlyActions = new LinkedList<>();
+    private final List<Object> remoteDatabasesOnlyActions = new LinkedList<>();
```

Comment:
Could you please investigate if its possible to use a more concrete typ here and above then object?
Using Object in a generic type is making the whole concept of generics ad absurdum.